gluten-ut/spark35/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -1132,6 +1132,7 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("SPARK-41017: filter pushdown with nondeterministic predicates")
   enableSuite[GlutenFileScanSuite]
   enableSuite[GlutenGeneratorFunctionSuite]
+    .exclude("SPARK-45171: Handle evaluated nondeterministic expression")
```

Comment:
Why do we add this test but also exclude it?

Reply:
the excluded test is spark side test not new added one, the new added one has `Gluten -` prefix